mirrord/config/src/network.rs

```diff
@@ -47,6 +47,23 @@ pub struct NetworkConfig {
     /// Resolve DNS via the remote pod.
     #[config(env = "MIRRORD_REMOTE_DNS", default = "true")]
     pub dns: bool,
+
+    // TODO(alex) [mid] 2022-11-17: Improve these docs.
```

Comment:
question - would we be able to generate mutual exclusive regexes to detect if more than 1 filter includes hte other? (for operator)

Reply:
seems we can do regex -> dfa -> reduce